scripts/xapi-domains.service

```diff
@@ -1,7 +1,8 @@
 [Unit]
 Description=Start/stop domains on dom0 start/shutdown
 Requires=proc-xen.mount
-After=remote-fs.target xapi.service
+Wants=xapi-init-complete.target
+After=remote-fs.target xapi-init-complete.target
```

Comment:
Will this still do the right thing on shutdown, i.e. stop xapi-domains before xapi?

Reply:
It doesn't change the shutdown behaviour. Since xapi-domains is started after xapi, it will be shut down before.